web/blueprints/user/__init__.py

```diff
@@ -377,17 +395,22 @@ def create():
     form = UserCreateForm()
     if form.validate_on_submit():
         try:
-            new_user = lib.user.move_in(name=form.name.data,
+            new_user, plain_password = lib.user.move_in(name=form.name.data,
                 login=form.login.data,
                 building=form.building.data, level=form.level.data,
                 room_number=form.room_number.data,
                 mac=form.mac.data,
                 processor=current_user,
                 email=form.email.data,
             )
+
+            pdf_data = b64encode(generate_user_sheet(new_user, plain_password)).decode('ascii')
+            pdf_storage = WebStorage(data=pdf_data, expiry=datetime.utcnow() + timedelta(minutes=15))
+            session.session.add(pdf_storage)
             session.session.commit()
 
-            flash(u'Benutzer angelegt', 'success')
+            flasksession['user_sheet'] = pdf_storage.id
+            flash(Markup(u'Benutzer angelegt. <a href="{}">Nutzerdatenblatt</a> verfügbar!'.format(url_for('.user_sheet'))), 'success')
```

Comment:
View functions should be a simple, replacable frontend (always think of the repetition that would occur when introducing the same functionality in a REST API).
Instead of doing things here like passing the binary data of the user sheet somewhere, this (including adding a new WebStorage object to the session) can be handled in a library function that returns the resulting storage object. The helper could be extended by a function that encapsulates the whole `b64-> ascii` encoding procedure.

Reply:
Don't use german strings please.